PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRawJetWithEP.h

```diff
@@ -184,6 +185,7 @@ class AliAnalysisTaskRawJetWithEP : public AliAnalysisTaskEmcalJet {
     Bool_t  fEPQA = kFALSE;           ///<
     Bool_t  fTrackQA = kFALSE;        ///<
     Bool_t  fBkgQA = kFALSE;          ///<
+    Bool_t  fSepEP = kFALSE;          ///<
```

Comment:
You need to increase your ClassDef - otherwise, this will have issues on the trains

Reply:
Thank you for advicing me about ClassDef.
I modified the number of streem